src/wireviz/wireviz.py

```diff
@@ -311,28 +312,28 @@ def main():
 
     args = parse_cmdline()
 
-    if not os.path.exists(args.input_file):
-        print(f'Error: input file {args.input_file} inaccessible or does not exist, check path')
+    file_in = Path(args.input_file)
+
+    if not file_in.exists():
+        print(f'Error: input file {file_in} inaccessible or does not exist, check path')
         sys.exit(1)
 
-    with open_file_read(args.input_file) as fh:
+    with open_file_read(file_in) as fh:
         yaml_input = fh.read()
 
     if args.prepend_file:
-        if not os.path.exists(args.prepend_file):
+        if not Path(args.prepend_file).exists():
             print(f'Error: prepend input file {args.prepend_file} inaccessible or does not exist, check path')
             sys.exit(1)
         with open_file_read(args.prepend_file) as fh:
             prepend = fh.read()
             yaml_input = prepend + yaml_input
 
     if not args.output_file:
```

Comment:
Consider swapping the `if` and the `else` statements and remove the `not`:
```suggestion
    if args.output_file:
```